### PR TITLE
:tada: Feature handing by team

### PR DESCRIPTION
**Pull request by niwinz**

### CODE REVIEW

File: backend/src/app/rpc/commands/files.clj [hunk 1]
```diff
@@ -183,26 +166,14 @@
 
 ;; --- HELPERS
 
-(defn get-team-id
-  [conn project-id]
-  (:team-id (db/get-by-id conn :project project-id {:columns [:team-id]})))
+;; (defn get-team-id
+;;   [conn project-id]
+;;   (:team-id (db/get-by-id conn :project project-id {:columns [:team-id]})))
```
Comment (hirunatan) [inline backend/src/app/rpc/commands/files.clj]: delete?

Comment (niwinz) [reply backend/src/app/rpc/commands/files.clj]: :fire: :+1:

File: backend/src/app/rpc/commands/teams.clj [hunk 2]
```diff
@@ -165,44 +199,54 @@
...
 
-(declare retrieve-users)
-(declare retrieve-team-for-file)
+(declare get-users)
+(declare get-team-for-file)
+
+;; (s/def ::get-team-users
+;;   (s/and (s/keys :req [::rpc/profile-id]
+;;                  :opt-un [::team-id ::file-id])
+;;          #(or (:team-id %) (:file-id %))))
```
Comment (hirunatan) [inline backend/src/app/rpc/commands/teams.clj]: delete?

File: backend/test/backend_tests/rpc_file_test.clj [hunk 3]
```diff
@@ -711,7 +718,8 @@
         (t/is (contains? result :file-id))
         (t/is (= (:id file) (:file-id result)))
         (t/is (nil? (get-in result [:page :objects frame1-id :thumbnail])))
-        (t/is (not= [] (get-in result [:page :objects frame1-id :shapes])))))
+        (t/is (not= [] (get-in result [:page :objects frame1-id :shapes]))))
+      )
```
Comment (hirunatan) [inline backend/test/backend_tests/rpc_file_test.clj]: parenthesis

File: backend/src/app/rpc/commands/management.clj [hunk 4]
```diff
@@ -296,26 +292,46 @@
     where rel.file_id = br.file_id
       and rel.library_file_id = br.library_file_id")
 
+(defn- check-teams-compatibility!
```
Comment (hirunatan) [inline backend/src/app/rpc/commands/management.clj]: Perhaps into app.common.features?

File: frontend/src/app/main/features.cljs [hunk 5]
```diff
@@ -5,103 +5,99 @@
...
-           (rx/map keyword)
-           (rx/map enable-feature))
+             ;; features (if *assert*
+             ;;            (into features
+             ;;                  (comp
+             ;;                   (filter #(not= % "components/v2"))
+             ;;                   (filter #(not= % "styles/v2"))
+             ;;                   (filter #(not (str/starts-with? % "fdata/"))))
+             ;;                  cfeat/supported-features)
+             ;;            features)
```
Comment (hirunatan) [inline frontend/src/app/main/features.cljs]: remove?

File: frontend/src/app/main/ui/workspace.cljs [hunk 6]
```diff
@@ -236,49 +235,3 @@
...
-                                              :project-id (:id project)}))]
-    (mf/use-effect
-     (fn []
-       #(st/emit! (dw/clear-remove-graphics))))
-
-    [:div.modal-overlay
-     [:div.modal-container.remove-graphics-dialog
-      [:div.modal-header
-       [:div.modal-header-title
-        [:h2 (tr "workspace.remove-graphics.title" (:file-name ctx))]]
```
Comment (hirunatan) [inline frontend/src/app/main/ui/workspace.cljs:259]: should remove those labels from .po files?

**Pull request merged**